src/matrix.rs

```diff
@@ -1074,6 +1141,16 @@ impl<S: BaseFloat> Transform<Point3<S>> for Matrix3<S> {
         Matrix3::look_at(dir, up)
     }
 
+    fn look_at_lh(eye: Point3<S>, center: Point3<S>, up: Vector3<S>) -> Matrix3<S> {
+        let dir = center - eye;
+        Matrix3::look_at_lh(dir, up)
+    }
+
+    fn look_at_rh(eye: Point3<S>, center: Point3<S>, up: Vector3<S>) -> Matrix3<S> {
```

Comment:
now I'm wondering - can we set up the indirection between 3 methods at the level of the `Transform` trait, as the default implementations?

Reply:
Perhaps. The overlap in functionality with the `Rotation` trait was making things a little confusing. There seems to be overlap between `Rotation`, `Transform`, and some of the functions on the struct impls themselves. 

I'm also trying to make things more consistent and settle on the following conventions:

`look_to_[lh|rh]` accepts a direction vector
`look_at_[lh|rh]` accepts a focus/center point